src/main/java/kg/peaksoft/giftlistb6/db/services/MailingListService.java

```diff
@@ -0,0 +1,65 @@
+package kg.peaksoft.giftlistb6.db.services;
+
+import kg.peaksoft.giftlistb6.db.models.MailingList;
+import kg.peaksoft.giftlistb6.db.repositories.MailingListRepository;
+import kg.peaksoft.giftlistb6.dto.requests.MailingListRequest;
+import kg.peaksoft.giftlistb6.dto.responses.AllMailingListResponse;
+import kg.peaksoft.giftlistb6.dto.responses.MailingListResponse;
+import kg.peaksoft.giftlistb6.exceptions.NotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import javax.transaction.Transactional;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class MailingListService {
+
+    private final MailingListRepository mailingListRepository;
+
+    public AllMailingListResponse saveMailingList(MailingListRequest request) {
+        MailingList mailingList = convertToEntity(request);
+        mailingListRepository.save(mailingList);
+        return convertToResponse(mailingList);
+    }
+
+    @Transactional
+    public MailingList convertToEntity(MailingListRequest request) {
+        MailingList mailingList = new MailingList();
+        mailingList.setName(request.getName());
+        mailingList.setPhoto(request.getImage());
+        mailingList.setText(request.getText());
+        mailingList.setCreateDate(LocalDateTime.now());
+        return mailingList;
+    }
+
+    @Transactional
+    public AllMailingListResponse convertToResponse(MailingList mailingList) {
+        AllMailingListResponse response = new AllMailingListResponse();
+        response.setId(mailingList.getId());
+        response.setName(mailingList.getName());
+        response.setImage(mailingList.getPhoto());
```

Comment:
Why setImage and getPhoto?

Reply:
because the entity was created with photo fields